src/modules/crud.js

```diff
@@ -0,0 +1,117 @@
+const taskContainer = document.querySelector('.taskList');
+const taskDesc = document.querySelector('.desc');
+const formInput = document.querySelector('.formInput');
```

Comment:
- _To maintain a best practice in Javascript, I recommend to you **avoid** global variables and use local variables:_